cpp/models/ode_secir/model.h

```diff
@@ -342,6 +347,33 @@ auto get_migration_factors(const Simulation<Base>& sim, double /*t*/, const Eige
     return factors;
 }
 
+template <class Base = mio::Simulation<Model>>
+auto test_commuters(Simulation<Base>& sim, Eigen::Ref<Eigen::VectorXd> migrated, double time)
+{
+    auto& model       = sim.get_model();
+    auto nondetection = 1.0;
```

Comment:
comment. What does nondetection do? and why 1.0?

Reply:
non detection is the share of commuters that are detected positive. If commuter detection has not started, no one is detected i.e. nondetection is 100%. This is implemented analogous to the secirvvs model.